test/controllers/api/v1/tickets_controller_test.rb

```diff
@@ -38,4 +38,17 @@ class Api::V1::TicketsControllerTest < ActionController::TestCase
     assert_response :success
   end
 
+  test 'should create ticket' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.